servicetalk-gradle-plugin-internal/src/main/groovy/io/servicetalk/gradle/plugin/internal/ServiceTalkLibraryPlugin.groovy

```diff
@@ -206,6 +207,18 @@ final class ServiceTalkLibraryPlugin extends ServiceTalkCorePlugin {
     }
   }
 
+  private static void applyPitestPlugin(Project project) {
+    project.configure(project) {
+      pluginManager.apply("info.solidsoft.pitest")
+
+      pitest {
```

Comment:
Gradle defines a group for each task. You can see them using `./gradlew tasks`.
What group this task belongs to? If it's not `verification`, consider doing something similar to pmd/spotbugs:

```
      tasks.withType(Pmd).all {
        group = "verification"
      }
```